media-gfx/phototonic/phototonic-2.1.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="An image viewer and organizer built with Qt and Exiv2"
+DESCRIPTION="Features
+* Support for common image formats and GIF animation
+* Supports tagging images, and filtering images by tags (IPTC)
+* Browse thumbnails recursively down a folder tree
+* Dynamic thumbnails loading
+* Image transformation and color manipulation
+* Display image information and metadata
+* Does not depend on any desktop environment."
+HOMEPAGE="https://github.com/oferkv/phototonic/"
+COPYRIGHT="2015-2018 Over Kashayov
+           2015-2018 Christopher roy
+           "
```

Comment:
```diff
-HOMEPAGE="https://github.com/oferkv/phototonic/"
+HOMEPAGE="https://github.com/oferkv/phototonic"
 COPYRIGHT="2015-2018 Over Kashayov
-           2015-2018 Christopher roy
-           "
+	2015-2018 Christopher Roy"
```

Reply:
Hmm, we have a paragraph in our wiki:
[A simple example (using the recommended style for a multi-line `COPYRIGHT`)](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#a-simple-example-using-the-recommended-style-for-a-multi-line-copyright)

Reply:
@fbrosson  edited... thank you..